### ISSUE TITLE
An error in LFU?

### ISSUE THREAD

**Issue by Liyunfan1998** [dgilland/cacheout#9]
```
from cacheout.lfu import LFUCache
cache = LFUCache(maxsize=4, ttl=0, timer=time.time, default=None)
for i in range(4):
    cache.add(1,1)
    cache.add(2,1)
for i in range(3):
    cache.add(3,1)
    cache.add(4,1)
cache.add(5,1)
cache.add(6,1)
cache.keys()
```

output should be "`odict_keys([1, 2, 4, 6])`"
but is instead "`odict_keys([1, 2, 3, 4, 6])`"

**Comment by dgilland** [comment]
This is indeed a bug. Thanks for reporting.

Will work on a fix.

**Comment by dgilland** [comment]
Fix in 235e6961e76ebe2619d93b4775657fd5320779f5 and released in [0.11.2](https://pypi.org/project/cacheout/0.11.2/).

**Comment by dgilland** [comment]
Thanks again for the bug report!
